library/ssl_tls12_server.c

```diff
@@ -2636,7 +2636,7 @@ static int ssl_get_ecdh_params_from_cert(mbedtls_ssl_context *ssl)
         case MBEDTLS_PK_ECKEY:
         case MBEDTLS_PK_ECKEY_DH:
         case MBEDTLS_PK_ECDSA:
-            key = mbedtls_pk_ec_ro(*pk);
+            key = mbedtls_pk_ec_rw(*pk);
```

Comment:
Ah, this came as a surprise to me, since it looks all we're doing here is importing the key to a PSA key slot. But the problem is that `mbedtls_ecp_write_key()` wants a non-`const` pointer for no reason other that we forgot to type `const` in its signature. Sigh.

(No changes required here.)

Reply:
Yes, there are some cases in which I had to go for the `rw` even if the usage was `ro` because otherwise I had also to change the public ECP function interface :/

Reply:
It's OK, anyway we're planning to remove those calls in the near future anyway. This one is #7406.